### PR TITLE
[ALLUXIO-1740] Remove NIO Code in Worker

### PR DESCRIPTION
**Pull request by calvinjia**
https://tachyon.atlassian.net/browse/ALLUXIO-1740

### CODE REVIEW

File: docs/_data/table/en/worker-configuration.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ alluxio.worker.data.port:
   The port Alluxio's worker's data server runs on.
 alluxio.worker.data.server.class:
   "Selects the networking stack to run the worker with. Valid options are:
-  `alluxio.worker.netty.NettyDataServer`, `alluxio.worker.nio.NIODataServer`."
+  `alluxio.worker.netty.NettyDataServer`.
```
Comment (aaudiber) [inline docs/_data/table/en/worker-configuration.yml]: not sure why some of these are in quotes, but there's a mismatched quote now

Comment (calvinjia) [inline docs/_data/table/en/worker-configuration.yml]: Good catch, updated.

**Comment by calvinjia** [comment]
Jenkins, test this please.

**Comment by aaudiber** [comment]
LGTM

File: docs/_data/table/cn/worker-configuration.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ alluxio.worker.data.folder:
 alluxio.worker.data.port:
   运行Alluxio worker数据服务的端口。
 alluxio.worker.data.server.class:
-  选择运行worker的网络栈，可选值为：`alluxio.worker.netty.NettyDataServer`和`alluxio.worker.nio.NIODataServer`。
+  选择运行worker的网络栈，可选值为：`alluxio.worker.netty.NettyDataServer`。
```
Comment (yupeng9) [inline docs/_data/table/cn/worker-configuration.yml]: Do we still need this abstraction given we only have one type of server?

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
LGTM

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**